src/library/themes/themeFromTokens.js

```diff
@@ -0,0 +1,52 @@
+/* @flow */
+import colorAliases from '../../../packages/mineral-ui-tokens/src/colorAliases';
```

Comment:
If you still wanted to add aliases for these, I think you would need to do the following.

1. Update your import paths. e.g.

```js
import colorAliases from 'mineral-ui-tokens/colorAliases';
```

Add a line to `.babelrc.js`

```js
'mineral-ui-tokens': './packages/mineral-ui-tokens/src'
```

Add a line to `.flowconfig`

```js
module.name_mapper='^mineral-ui-tokens\/?\(.*\)$' -> '<PROJECT_ROOT>/packages/mineral-ui-tokens/src/\1'
```